torch/linalg/__init__.py

```diff
@@ -204,31 +213,42 @@
 slogdet = _add_docstr(_linalg.linalg_slogdet, r"""
 linalg.slogdet(input, *, out=None) -> (Tensor, Tensor)
 
-Calculates the sign and natural logarithm of the absolute value of a square matrix's determinant,
-or of the absolute values of the determinants of a batch of square matrices :attr:`input`.
-The determinant can be computed with ``sign * exp(logabsdet)``.
+Computes the sign and logarithm of the norm of the determinant of a square matrix or batch of square matrices.
 
-Supports input of float, double, cfloat and cdouble datatypes.
+For a complex :attr:`input`, as the determinant is complex in this case, it returns the angle
```

Comment:
This is confusing. Is this disagreeing with the first sentence or elaborating on a mathematical perspective? If providing a mathematical perspective then the paragraph should explain it's providing a perspective and possibly be put into a note.

Reply:
It sort of "disagrees". This function is tricky, as for real inputs it's natural to talk about the "sign of the determinant". For complex inputs, that concept of sign turns into the concept of angle. 
It makes sense to put the two cases separately, even though "sign" the real case is just the complex case on real matrices. Now, if we put the two on the same line, it'd be too long of a "short description". I am not sure how to marry all these things really.